modules/sync/syncs/httpsynchronization.cpp

```diff
@@ -134,6 +158,82 @@ std::string HttpSynchronization::generateUid() {
     return fmt::format("{}/{}", _identifier, _version);
 }
 
+void HttpSynchronization::createSyncFile(bool fullySynchronized = true) const {
```

Comment:
For Variable name, see above


Also, this line looks to me as if if it is too long, but not sure